@navikt/core/react/src/alert/base-alert/close/BaseAlertClose.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef } from "react";
+import { XMarkIcon } from "@navikt/aksel-icons";
+import { Button } from "../../../button";
+import { useRenameCSS } from "../../../theme/Theme";
+import { useI18n } from "../../../util/i18n/i18n.hooks";
+import { useBaseAlert } from "../root/BaseAlertRoot.context";
+
+type BaseAlertCloseProps = React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+/**
+ * @see 🏷️ {@link BaseAlertCloseProps}
+ * @example
+ * ```jsx
+ *  <BaseAlert>
+ *    <BaseAlert.Header>
+ *      <BaseAlert.Title>Info tittel</BaseAlert.Title>
+ *      <BaseAlert.Close aria-label="Lukk varsel">
```

Comment:
Should we have `aria-label` in the example when we set a title on the icon by default anyways?

Also, the tag is not closed.

```suggestion
 *      <BaseAlert.Close onClick={...} />
```